.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
[According to documentation](https://docs.npmjs.com/files/package-lock.json), package-lock.json.json "is intended to be committed into source repositories, and therefore should not be git ignored".